network/p2p/sender.go

```diff
@@ -0,0 +1,210 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package p2p
+
+import (
+	"context"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/snow/engine/common"
+	"github.com/ava-labs/avalanchego/utils/set"
+)
+
+var (
+	_ AppSender = (*Sender)(nil)
```

Comment:
q: what is the Sender structure for? Can't we just use common.Sender instead of it?